frontend/src/components/Recipe/ContextMenu.vue

```diff
@@ -88,6 +92,9 @@ export default {
         ...this.defaultMenu,
       ];
     },
+    recipeText() {
+      return this.$t(`I wanted to share you my {0} recipe.`, [this.name]);
```

Comment:
You're not using Vue-i18n strings correctly. https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting
You need to declare new strings in frontend/src/locales/messages/en-US.json, and use the key you defined there when you call $t. Otherwise the string will never be translated.
Have a look at other calls of $t and check out the file I mentioned for an example.

Reply:
I didn't Know how to do this and used the advice from the discord guys. Will try to figure it out